docs/HTTP_API.md

```diff
@@ -65,8 +65,7 @@
     The `NoopAuthManager` accepts all authentication requests,
     regardless of the existence or contents of any
     `Authorization` header.
-    For more details, see
-    [`README.md`](https://github.com/cryostatio/cryostat#user-authentication--authorization)).
+    For more details, see [`README.md`](../README.md)).
```

Comment:
There is an extra closing bracket here :D

Reply:
I think we can use `../README.md#user-authentication--authorization` to point that that section.